tests/test/test_export.js

```diff
@@ -592,7 +592,7 @@ describe('Exporter', function () {
           "bounds": {
             "x": 1,
             "y": 1,
-            "z": 1
+            "z": 4
```

Comment:
This seems a little weird, is this intentional?

Reply:
I modified the test to make sure that we are doing the axis swapping right.

Reply:
Ah, okay.  All good then.